planning/planning_scene_monitor/include/moveit/planning_scene_monitor/planning_scene_monitor.h

```diff
@@ -521,6 +532,7 @@ class PlanningSceneMonitor : private boost::noncopyable
   ros::CallbackQueue                    callback_queue_;
   boost::scoped_ptr<ros::AsyncSpinner>  spinner_;
   ros::Time                             last_robot_motion_time_; /// Last time the robot has moved
+  bool                                  enforce_next_state_update_;
```

Comment:
You only ever *assign* `enforce_next_state_update_` in this request, but never read from it.
This doesn't make sense.